agent/src/index.ts

```diff
@@ -63,7 +63,7 @@ import { zksyncEraPlugin } from "@elizaos/plugin-zksync-era";
 import { cronosZkEVMPlugin } from "@elizaos/plugin-cronoszkevm";
 import { abstractPlugin } from "@elizaos/plugin-abstract";
 import { avalanchePlugin } from "@elizaos/plugin-avalanche";
-import { webSearchPlugin } from "@elizaos/plugin-web-search";
```

Comment:
removed a plugin

Reply:
Some one already put it in line 10, so this one should be fine